### ISSUE TITLE
Getting error in update_feedback

### ISSUE THREAD

**Issue by pribadihcr** [langchain-ai/langsmith-cookbook#134]
Hi, I got getting error when update the feedback:

```
feedback_ = client.create_feedback(request.run_id, "user_score", score=int(request.score))
    feedback_update = {
                "comment": request.comment,
                "correction": {"desired": request.correction},
                "feedback_id": feedback_.id,
            }
client.update_feedback(**feedback_update)
```

ValueError: ('404 Client Error: Not Found for url: https://api.smith.langchain.com/feedback/e8430657cb024c2ab210e804f7527432', '{"detail":"Resource not found"}')

**Comment by hinthornw** [comment]
Hi @pribadihcr thank you for raising!

tl;dr - 
1. Update to sdk version 0.0.49
2. Set "eager" to True when creating feedback: `client.create_feedback(request.run_id, "user_score", score=int(request.score), eager=True)` 


Basically in order to better scale to high throughput use cases, runs and associated feedback are enqueued prior to committing to the database. Patch requests by default bypass this queue. To bypass the queue for the feedback write request, you can set eager to True. 

Note that in this case, writing the feedback still will require the run itself to be present in the DB. The create_feedback method handles will retry (in case the run is still in the queue by the time the user submits feedback), but since it is a sync call, you may want to consider wrapping in asyncio
